Dockerfile

```diff
@@ -56,22 +56,31 @@ RUN \
   tar -xzf ${APP_NAME}.tar.gz && \
   rm ${APP_NAME}.tar.gz
 
-FROM dkr.plural.sh/plural/plural-cli:0.3.12 as cmd
+FROM gcr.io/pluralsh/alpine:3 as tools
 
-FROM gcr.io/pluralsh/alpine:3 as helm
+# renovate: datasource=github-releases depName=helm/helm
+ENV HELM_VERSION=v3.7.0
 
-ARG VERSION=3.7.0
-ENV TERRAFORM_VERSION=0.15.2
+# renovate: datasource=github-releases depName=hashicorp/terraform
+ENV TERRAFORM_VERSION=v0.15.2
 
-# ENV BASE_URL="https://storage.googleapis.com/kubernetes-helm"
-ENV BASE_URL="https://get.helm.sh"
-ENV TAR_FILE="helm-v${VERSION}-linux-amd64.tar.gz"
+# renovate: datasource=github-releases depName=pluralsh/plural-cli
+ENV CLI_VERSION=v0.4.3
+
+# renovate: datasource=github-tags depName=kubernetes/kubectl
+ENV KUBECTL_VERSION=v1.16.14
 
 RUN apk add --update --no-cache curl ca-certificates unzip wget openssl build-base && \
-    curl -L ${BASE_URL}/${TAR_FILE} | tar xvz && \
+    curl -L https://get.helm.sh/helm-${HELM_VERSION}-linux-amd64.tar.gz | tar xvz && \
     mv linux-amd64/helm /usr/local/bin/helm && \
-    wget https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip && \
-    unzip terraform_${TERRAFORM_VERSION}_linux_amd64.zip -d /usr/local/bin && \
+    wget https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION/v/}/terraform_${TERRAFORM_VERSION/v/}_linux_amd64.zip && \
+    unzip terraform_${TERRAFORM_VERSION/v/}_linux_amd64.zip -d /usr/local/bin && \
+    curl -L https://github.com/pluralsh/plural-cli/releases/download/${CLI_VERSION}/plural-cli_${CLI_VERSION/v/}_Linux_x86_64.tar.gz | tar xvz plural && \
```

Comment:
that removes the leading `v` from the ENV var version. This is needed for Renovate since the version it will get from the GitHub release will have the format `vX.Y.Z`, but in the download URL we need the format `X.Y.Z`.